### PR TITLE
1011 Create Windows installer

### PR DESCRIPTION
**Pull request by jperedadnr**
Create Windows installer

## Description

Fixes pyrsia/pyrsia#1011

This PR creates a Windows installer (MSI) for the binaries `pyrsia` (CLI) and `pyrsia_node` (service).

The MSI installer is created with Wix, which uses a couple of PowerShell scripts to create and start the Windows service. 

For that purpose, [`scoop`](https://scoop.sh) is used as a Windows package manager, to install [`nssm`](https://nssm.cc), the service manager that is finally used to deal with the `pyrsia_node` executable. In case any of those is missing, the script will install them. 

The MSI installer can be executed by double clicking on it, but it is more convenient to install it from a terminal, with non-admin rights, like:

```
msiexec /i C:\Users\$user\Downloads\pyrsia.msi /L*v C:\tmp\log.txt ROOTDRIVE="C:"
```

That will generate `C:\Pyrsia\Pyrsia` with
```
+ bin
   - pyrsia.exe
+ service
   - pyrsia_node.exe
   - InstallPyrsiaNodeAsWindowsService.ps1
   - InstallNSSM.ps1
+ Readme.txt
```

and a log file `C:\tmp\log.txt` with the log of the process.

The `pyrsia` CLI is added to the system PATH. For instance, to test it run:
```
>pyrsia -s
Connected Peers Count:       1
```

Also, as the service is installed and started, if everything works as expected, the files under `C:\Pyrsia\Pyrsia` should be:
```
+ bin
   - pyrsia.exe
+ service
   - pyrsia_node.exe
   - InstallPyrsiaNodeAsWindowsService.ps1
   - InstallNSSM.ps1
   - pyrsia_logs.txt
   + pyrsia
      -  p2p_keypair.ser
+ Readme.txt
```

where the pyrsia_logs file should show the expected:

```
 2022-09-29T17:47:47.335Z DEBUG pyrsia_node > Parse CLI arguments
 2022-09-29T17:47:47.335Z DEBUG pyrsia_node > Create p2p components
 2022-09-29T17:47:47.345Z DEBUG pyrsia_node > Start p2p event loop
 2022-09-29T17:47:47.345Z DEBUG pyrsia_node > Create blockchain service component
 2022-09-29T17:47:47.346Z DEBUG pyrsia_node > Create transparency log service
 2022-09-29T17:47:47.346Z DEBUG pyrsia_node > Create pyrsia services
 2022-09-29T17:47:47.346Z DEBUG pyrsia_node > Create build event client
 2022-09-29T17:47:47.346Z DEBUG pyrsia_node > Create artifact service
 2022-09-29T17:47:47.346Z DEBUG pyrsia_node > Create build service
 2022-09-29T17:47:47.347Z DEBUG pyrsia_node > Create verification service
 2022-09-29T17:47:47.347Z DEBUG pyrsia_node > Start build event loop
 2022-09-29T17:47:47.347Z DEBUG pyrsia_node > Setup HTTP server
 2022-09-29T17:47:47.347Z DEBUG pyrsia_node > Pyrsia Docker Node will bind to host = 127.0.0.1, port = 7888
 2022-09-29T17:47:47.347Z DEBUG pyrsia_node > Setup HTTP routing
 2022-09-29T17:47:47.347Z DEBUG pyrsia_node > Setup HTTP server
 2022-09-29T17:47:47.348Z INFO  pyrsia_node > Pyrsia Docker Node will start running on 127.0.0.1:7888
 2022-09-29T17:47:47.348Z DEBUG pyrsia_node > Start p2p components
 2022-09-29T17:47:47.348Z DEBUG pyrsia::network::client > p2p::Client::listen "/ip4/0.0.0.0/tcp/0"
 2022-09-29T17:47:47.349Z INFO  pyrsia_node             > Looking up bootstrap node
2022-09-29T17:47:47.352Z INFO  pyrsia::network::event_loop > Local node is listening on "/ip4/192.168.68.117/tcp/64994/p2p/12D3KooWSu5Zz38GgM1L1BTFoBawJqYWdCtWJs9fH3RWyj7TTr61"
```

Publishing the MSI 

So far the MSI is built and published as an artifact that is publicly available at the bottom of the workflow summary page.
For instance, and for testing purposes while this PR is merged, this [workflow](https://github.com/pyrsia/pyrsia/actions/runs/3154860402) has an artifact that can be used: https://github.com/pyrsia/pyrsia/suites/8542521320/artifacts/381257594
 
As a follow-up PR, we should publish the installer following the same procedure as the debian one: https://github.com/pyrsia/pyrsia/issues/1119.

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've built the code `cargo build --all-targets` successfully.
- [x] I've run the unit tests `cargo test --workspace` and everything passes.
- [x] I've made sure my rust toolchain is current `rustup update`.

### CODE REVIEW

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Review by fishseabowl — DISMISSED** [review]

**Review by fishseabowl — DISMISSED** [review]

File: installers/windows/Readme.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Pyrsia installer files for Windows
+
+How to install the msi:
+
+Open a non-elevated terminal and type:
```
[the remark this answers is not in the document]
Comment (jperedadnr) [reply installers/windows/Readme.txt]: Thanks for your review, I've provided a much more detailed readme now.

**Review by fishseabowl — DISMISSED** [review]

**Comment by erwin1** [comment]
@sbtaylor15 I think you can re-approve after @jperedadnr's changes. there will be 2 outputs: a pyrsia.msi and a pyrsia-service.msi which we should both make available for download. see #1119

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Pull request merged**